scriptspony/templates/index.mak

```diff
@@ -23,8 +23,16 @@ from scripts import auth
 	  </td>
           %if host not in (locker+'.scripts.mit.edu',):
             <td class="nbr"><a href="${tg.url('/edit/'+locker+'/'+host)}" class="btn sm-btn" aria-label="Edit"><span class="fa fa-pencil" aria-hidden="true"></span></a></td>
+              %if has_cert:
+                  <td><span class="secure-btn-disabled"><span class="fa fa-lock"></span></span></td>
+              %elif (not host.endswith('.mit.edu')) or auth.on_scripts_team():
+                  <td class="nbr"><a href="${tg.url('/request_cert/'+locker+'/'+host)}" class="btn sm-btn" aria-label="Add HTTPS"><span class="fa fa-plus"></span></a></td>
+              %else:
+                  <td><span class="edit-btn-disabled"><span class="fa fa-ban"></span></span></td>
```

Comment:
This should link to the FAQ rather than suggesting that certificates on mit.edu are impossible.